documentation/experimental_ui.md

```diff
@@ -0,0 +1,78 @@
+# Experimental UI
+---
+
+We believe that web scraping is a process. It might seem easy to extract first 
```

Comment:
The [CrawlyUI project] (link to repo) is an additional opt-in user interface for managing and developing Crawly spiders.

Don't need to repeat the philosophy of crawly

Reply:
docs should be short and to the point, cos people will be skimming through them